glide-core/src/ffi/ffi_client.rs

```diff
@@ -0,0 +1,472 @@
+        RequestType::IncrBy => Some(cmd("INCRBY")),
+        RequestType::IncrByFloat => Some(cmd("INCRBYFLOAT")),
+        RequestType::Decr => Some(cmd("DECR")),
+        RequestType::DecrBy => Some(cmd("DECRBY")),
+        RequestType::HashGetAll => Some(cmd("HGETALL")),
+        RequestType::HashMSet => Some(cmd("HMSET")),
+        RequestType::HashMGet => Some(cmd("HMGET")),
+        RequestType::HashIncrBy => Some(cmd("HINCRBY")),
+        RequestType::HashIncrByFloat => Some(cmd("HINCRBYFLOAT")),
+        RequestType::LPush => Some(cmd("LPUSH")),
+        RequestType::LPop => Some(cmd("LPOP")),
+        RequestType::RPush => Some(cmd("RPUSH")),
+        RequestType::RPop => Some(cmd("RPOP")),
+        RequestType::LLen => Some(cmd("LLEN")),
+        RequestType::LRem => Some(cmd("LREM")),
+        RequestType::LRange => Some(cmd("LRANGE")),
+        RequestType::LTrim => Some(cmd("LTRIM")),
+        RequestType::SAdd => Some(cmd("SADD")),
+        RequestType::SRem => Some(cmd("SREM")),
+        RequestType::SMembers => Some(cmd("SMEMBERS")),
+        RequestType::SCard => Some(cmd("SCARD")),
+        RequestType::PExpireAt => Some(cmd("PEXPIREAT")),
+        RequestType::PExpire => Some(cmd("PEXPIRE")),
+        RequestType::ExpireAt => Some(cmd("EXPIREAT")),
+        RequestType::Exists => Some(cmd("EXISTS")),
+        RequestType::Unlink => Some(cmd("UNLINK")),
+        RequestType::TTL => Some(cmd("TTL")),
+        RequestType::Zadd => Some(cmd("ZADD")),
+        RequestType::Zrem => Some(cmd("ZREM")),
+        RequestType::Zrange => Some(cmd("ZRANGE")),
+        RequestType::Zcard => Some(cmd("ZCARD")),
+        RequestType::Zcount => Some(cmd("ZCOUNT")),
+        RequestType::ZIncrBy => Some(cmd("ZINCRBY")),
+        RequestType::ZScore => Some(cmd("ZSCORE")),
+        RequestType::Type => Some(cmd("TYPE")),
+        RequestType::HLen => Some(cmd("HLEN")),
+        RequestType::Echo => Some(cmd("ECHO")),
+        RequestType::ZPopMin => Some(cmd("ZPOPMIN")),
+        RequestType::Strlen => Some(cmd("STRLEN")),
+        RequestType::Lindex => Some(cmd("LINDEX")),
+        RequestType::ZPopMax => Some(cmd("ZPOPMAX")),
+        RequestType::XAck => Some(cmd("XACK")),
+        RequestType::XAdd => Some(cmd("XADD")),
+        RequestType::XReadGroup => Some(cmd("XREADGROUP")),
+        RequestType::XRead => Some(cmd("XREAD")),
+        RequestType::XGroupCreate => Some(get_two_word_command("XGROUP", "CREATE")),
+        RequestType::XGroupDestroy => Some(get_two_word_command("XGROUP", "DESTROY")),
+        RequestType::XTrim => Some(cmd("XTRIM")),
+        RequestType::HSetNX => Some(cmd("HSETNX")),
+        RequestType::SIsMember => Some(cmd("SISMEMBER")),
+        RequestType::Hvals => Some(cmd("HVALS")),
+        RequestType::PTTL => Some(cmd("PTTL")),
+        RequestType::ZRemRangeByRank => Some(cmd("ZREMRANGEBYRANK")),
+        RequestType::Persist => Some(cmd("PERSIST")),
+        _ => None,
```

Comment:
If this case happens, this is considered UB, even if we try to switch this to a non-exhaustive enum. See here: https://www.reddit.com/r/rust/comments/cfp6lt/non_exhaustive_and_ffi/

We should probably just mark this as a safety condition in the documentation.

Reply:
or follow the good advice from the link you provided, and implement TryFrom for the enum.